django/core/validators.py

```diff
@@ -394,33 +394,40 @@ def compare(self, a, b):
 
 @deconstructible
 class StepValueValidator(BaseValidator):
-    message = _(
-        "Ensure this value is a multiple of step size %(limit_value)s, starting from "
-        "%(offset)s, e.g. %(offset)s, %(valid_value1)s, %(valid_value2)s, and so on."
-    )
+    message = _("Ensure this value is a multiple of step size %(limit_value)s.")
     code = "step_size"
 
     def __init__(self, limit_value, message=None, offset=None):
         super().__init__(limit_value, message)
-        self.offset = 0 if offset is None else offset
+        if offset is not None:
+            self.message = _(
+                "Ensure this value is a multiple of step size %(limit_value)s, "
+                "starting from %(offset)s, e.g. %(offset)s, %(valid_value1)s, "
+                "%(valid_value2)s, and so on."
```

Comment:
```              
                "Ensure this value is a multiple of step size %(limit_value)s, "
                "remainder %(offset)s."
```
Or "with an offset of "? We had no mentions of `min_value` and `max_value` before as they have their own error messages so can still be communicated to the user if required.

Reply:
`remainder`/`offset` may not be clear to most users.

Reply:
Actually I think it's ok, it is explaining the calculation rather than necessarily giving values the user can use. :+1: